### DISCUSSION TITLE
Is there a way to verify WSL's integrity once installed?

### DISCUSSION THREAD

**Discussion by dev-0-null** [microsoft/WSL#9343] [Q&A] [answered]
It is the only service listed as unsigned in autoruns, so I submitted it to VirusTotal and under signature verification it is indeed listed as unsigned. 

For comparison, I submitted MicrosoftEdgeUpdate.exe and it has signature verification and is listed as "distributed by Microsoft."

Essentially, is there a way to verify the integrity of WSL once installed, or is it normally unsigned?

Note: I did install both Sysinternals Suite and WSL from the Microsoft Store.

**Comment by Biswa96** [comment, marked as answer]
See this thread https://github.com/microsoft/WSL/issues/9159
